docs/content/docs/patterns/observability/tracing.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Tracing"
+description: "Tracing"
+type: docs
+---
+
+# Tracing
+Open Service Mesh (OSM) allows optional deployment of Jaeger for tracing. 
+
+## Jaeger
+[Jaeger](https://www.jaegertracing.io/) is an open source tracing system used for monitoring and troubleshooting distributed systems. It can be deployed with OSM as a new instance or you may bring your own instance. 
+
+### Automatic bring up
+By default, Jaeger deployment and tracing as a whole is disabled.
+
+Jaeger can be automatically deployed by using the `--deploy-jaeger` OSM CLI flag at install time or by toggling the `deployJaeger` value in `values.yaml`.
+
+Similarly, tracing can be enabled and customized during installation (`tracing` section in `values.yaml`) or at runtime by editing the `osm-config` ConfigMap. Tracing can be enabled, disabled and configured at any time to support BYO scenarios.
+
+### BYO (Bring-your-own)
+The following section documents the additional steps needed to allow an already running instance of Jaeger to integrate with your OSM control plane.
+
+#### Prerequisites
+* A running Jaeger instance
+
+The sections below outline how to make required updates depending on whether you already have a running instance of OSM or are installing OSM for the first time. In either case, the following `tracing` values in `values.yaml` are being updated to point to your Jaeger instance:
+1. `enable`: set to `true` to tell the Envoy connection manager to send tracing data to a specific address (cluster)
+1. `address`: set to the destination cluster of your Jaeger instance
+1. `port`: set to the destination port for the listener that you intend to use
+1. `endpoint`: set to the destination's API or collector endpoint where the spans will be sent to
+
+
+#### Deploying Jaeger to a running instance of the OSM control plane
+
+If you already have OSM running, `tracing` values must be updated in the OSM ConfigMap using:
+
+```console
+osm mesh upgrade --enable-tracing --tracing-address <tracing server hostname> --tracing-port <tracing server port> --tracing-endpoint <tracing server endpoint>
+```
+
+You can verify these changes have been deployed by inspecting `osm-config`:
+```console
+kubectl get configmap osm-config -n osm-system -o yaml
+```
+
+#### Deploying Jaeger during OSM deployment
+
+To deploy Jaeger during OSM installation, you can use the `--set` flag as shown below to update the values:
+
+```console
+osm install --set OpenServiceMesh.tracing.enable=true,OpenServiceMesh.tracing.address=<tracing server hostname>,OpenServiceMesh.tracing.port=<tracing server port>,OpenServiceMesh.tracing.endpoint=<tracing server endpoint>
```

Comment:
If the user is using the osm cli to install Jaeger, do they still need to provide the address, port, endpoint or will the default values in the values.yaml work as expected?

Reply:
If they're using the osm CLI with the `--deploy-jaeger` flag, wouldn't that fall under automatic bring up? I do see at least the default `port` value being used in [jaeger-service.yaml](https://github.com/openservicemesh/osm/blob/main/charts/osm/templates/jaeger-service.yaml) and [jaeger-deployment.yaml](https://github.com/openservicemesh/osm/blob/main/charts/osm/templates/jaeger-deployment.yaml). I can include a note for clarity if necessary?
cc: @eduser25 may know be better able to answer this

Reply:
@SanyaKochhar Can you pls make this section more clear that these flags are only needed for BYO Jaeger/tracing solution, NOT for when --deploy-jaeger flag is used.